### ISSUE TITLE
Not working with MXNet nightly build (1.5.0b20181222)

### ISSUE THREAD

**Issue by YutingZhang** [wkcn/MobulaOP#24]
With MXNet 1.5 nightly build (1.5.0b20181222), `import mobula` gives segmentation fault.

**Comment by wkcn** [comment]
Thanks for your report. I will check it.

**Comment by YutingZhang** [comment]
Thank you for your reply.
EC2 machine: p3.16xlarge 
OS: Ubuntu 16.04
CUDA: 9.0
Python: 3.6.6 (anaconda mxnet_p36 from AWS deep learning AMI)
MXNet: installed by `pip install -U mxnet-cu90mkl --pre` (so now it is mxnet-cu90mkl-1.5.0b20181228)

Error message:

```
[15:47:50] $ python3 -c 'import mobula'

Segmentation fault: 11

Stack trace returned 10 entries:
[bt] (0) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/mxnet/libmxnet.so(+0x3ebbea) [0x7f20fb0ecbea]
[bt] (1) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/mxnet/libmxnet.so(+0x34740b6) [0x7f20fe1750b6]
[bt] (2) /lib/x86_64-linux-gnu/libc.so.6(+0x354b0) [0x7f213dd524b0]
[bt] (3) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/torch/_C.cpython-36m-x86_64-linux-gnu.so(+0x420308) [0x7f206b1c1308]
[bt] (4) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/torch/_C.cpython-36m-x86_64-linux-gnu.so(+0x422520) [0x7f206b1c3520]
[bt] (5) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/torch/_C.cpython-36m-x86_64-linux-gnu.so(+0xbbfcb2) [0x7f206b960cb2]
[bt] (6) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/torch/_C.cpython-36m-x86_64-linux-gnu.so(torch::onnx::initONNXBindings(_object*)+0x4f) [0x7f206b95aeff]
[bt] (7) /home/ubuntu/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages/torch/_C.cpython-36m-x86_64-linux-gnu.so(PyInit__C+0x15a) [0x7f206b11f60a]
[bt] (8) python3(_PyImport_LoadDynamicModuleWithSpec+0x185) [0x55c473b50b85]
[bt] (9) python3(+0x215d85) [0x55c473b50d85]
```

**Comment by wkcn** [comment]
I create a new branch [patch_for_24](https://github.com/wkcn/MobulaOP/tree/patch_for_24) to fix the bug.
It may work. Thanks.

**Comment by YutingZhang** [comment]
@wkcn Thank you so much! The patch is working.

**Comment by wkcn** [comment]
Fixed the bug in PR #26 :)
